pkg/mlsstore/models.go

```diff
@@ -0,0 +1,24 @@
+package mlsstore
+
+import "github.com/uptrace/bun"
+
+type Installation struct {
+	bun.BaseModel `bun:"table:installations"`
+
+	ID            string `bun:",pk"`
+	WalletAddress string `bun:"wallet_address,notnull"`
+	CreatedAt     int64  `bun:"created_at,notnull"`
+	RevokedAt     *int64 `bun:"revoked_at"`
+}
```

Comment:
@neekolas Any reason we're storing the derived state (when an installation was created/revoked) rather than the registration/revocation payloads that were originally used to perform those actions? I think it's important that the client can validate this data themselves